packages/dds/tree/src/feature-libraries/modular-schema/comparison.ts

```diff
@@ -203,6 +213,192 @@ export function allowsRepoSuperset(
 	return true;
 }
 
+/**
+ * @returns the discrepancies between two TreeStoredSchema objects
+ */
+export function allowsDocumentConcepts(
+	original: TreeStoredSchema,
```

Comment:
I think given the return type of `Incompatibility[]`, I named this function poorly. `allowsDocumentConcepts` was thinking more along the lines of `allowsRepoSuperset` / `allowsTreeSuperset`. Instead, this is probably more appropriate:

`getAllowedContentIncompatibilities(view, stored): Incompatibility[]`

Because the return type is already in terms of view / stored, we should make the parameters view & stored as well.

Reply:
An alternative direction you could take this in would be to change the structure of the return type so that you can express concepts that are only supported by one schema but not the other. Then this version of the function could be implemented much how we do `allowsRepoSuperset(a, b) && allowsRepoSuperset(b, a)`.

I think that naming & documenting semantics of fields on that type might be more complex than just keeping this signature though, so I'm ok with the suggestion on the first comment here.